src/app/find-help/accommodation-search/accommodation-search-form/accommodation-search-form.component.html

```diff
@@ -30,11 +30,21 @@
           </mat-error>
         </mat-form-field>
       </div>
-      <div class="col-md-4 search">
-        <button type="submit" class="btn btn-dark" [disabled]="form.invalid">
-          <mat-icon fontSet="material-icons-outlined">search</mat-icon>
-          <span>{{ "SEARCH" | translate }}</span>
-        </button>
+      <div class="form-search-wrapper col-md-4">
+        <div class="d-flex flex-column align-items-center ms-4">
+          <button type="submit" class="btn btn-dark" [disabled]="form.invalid">
+            <mat-icon fontSet="material-icons-outlined">search</mat-icon>
+            <span>{{ "SEARCH" | translate }}</span>
+          </button>
+          <button
+            [ngStyle]="{ visibility: showClearBtn ? 'visible' : 'hidden' }"
```

Comment:
ogólnie nie podoba mi się to rozwiązanie, z tego względu, że w mojej opinii UXowo nie jest to najlepsze rozwiązanie. Proponowałbym aby button był widoczny zawsze i sterowanie klasą `visually-disabled` - tak aby button był dostępny dla czytników. Do przegadania z UX